src/main/resources/db/changelog/init/15062021_create_location_table.yml

```diff
@@ -1,7 +1,7 @@
 databaseChangeLog:
   - changeSet:
       id: 15062021_create_location_table
-      author: volodymyr.bushko@gmail.com
+      author: volodymyr.bushko@gmail.com & hreshchuk@gmail.com
```

Comment:
don't need to add new author
initial is enough. all other changes can be viewed via Git

Reply:
Removed hresh...